tasks/install-docker.yml

```diff
@@ -77,3 +66,14 @@
   become_user: "{{ work_user }}"
   command: open --background -a Docker
   changed_when: false
+
+- name: Change docker-compose settings
```

Comment:
This is docker settings.
Try replace to `"credsStore" : "desktop"`

Reply:
This setting is by default and just doesn't work with it, therefore I delete it